src/plugins/health.ts

```diff
@@ -1,60 +1,156 @@
 import {Plugin, Cordova} from './plugin';
 
-/**
- * @param: startDate: {type: Date}, start date from which to get data
- * @param: endDate: {type: Date}, end data to which to get the data
- * @param: dataType: {type: String}, the data type to be queried (see below for supported data types)
- */
+/*
```

Comment:
This large block of documentation is not necessary. It won't be visible on the site anyways.

We can add a link in the main documentation block telling people to visit the plugin's repo to learn more about the valid data types. Or this table can be under `@advanced` tag. Example:

```
/**
* @name PluginName
* @description ...
* @usage ...
* @advanced
* Put the table here 
*/
```